database/seeders/OtherDataSeeder.php

```diff
@@ -24,159 +25,116 @@ class OtherDataSeeder extends Seeder
-            ]);
-
-            echo "\e[33mCreating:\e[0m  ".$index.'/'.$n." committee memberships\r";
-        }
-
-        $time_end = microtime(true);
-        echo PHP_EOL."\e[32mCreated:\e[0m   ".$n.' committee memberships '.'('.round(($time_end - $time_start), 2).'s)'.PHP_EOL;
+        $output->task('creating achievement ownerships', function () use ($members) {
+            foreach($members as $member){
+                AchievementOwnership::factory()
+                    ->count(fake()->randomDigit())
+                    ->for($member)
+                    ->create();
+            }
+        });
 
         // Create activity participations
-        echo "\e[33mCreating:\e[0m  activity participations".PHP_EOL;
-        $time_start = microtime(true);
-
-        foreach (Activity::orderBy('id', 'desc')->take(25)->get() as $activity) {
-            if (! $activity->event) {
-                continue;
+        $output->task('creating activity participations', function () use ($members) {
+            $activities = Activity::has('event')->orderBy('id', 'desc')->take(25)->get();
+            foreach($activities as $activity) {
+                foreach($members->random(fake()->numberBetween(0, $members->count())) as $member) {
+                    ActivityParticipation::factory()->for($activity)->for($member);
+                }
             }
+        });
 
-            $mintime = date('U', strtotime($activity->registration_start));
-            $maxtime = date('U', strtotime($activity->event->start));
-
-            $offset = mt_rand(-15, 5);
-            $max = ($activity->participants < 0 ? mt_rand(0, 50) : $activity->participants);
-            $p = (max($max + $offset, 0));
-
-            $j = 0;
-            for ($i = 0; $i < $p; $i++) {
-                $startDate = date('U', ($maxtime > $mintime ? mt_rand($mintime, $maxtime) : $mintime));
-                $endDate = date('Y-m-d H:i:s', ($maxtime > $startDate ? mt_rand($startDate, $maxtime) : $maxtime));
+        // Create pages
+        $n = 10;
+        $output->task("creating $n pages", fn () => Page::factory()->count($n)->create());
 
-                ActivityParticipation::create([
-                    'user_id' => array_random($users),
-                    'activity_id' => $activity->id,
-                    'created_at' => date('Y-m-d H:i:s', $startDate),
-                    'deleted_at' => (mt_rand(1, 3) == 1 ? $endDate : null),
-                    'backup' => ($activity->participants != -1 && $j > $activity->participants),
-                ]);
-
-                $j++;
-            }
-        }
-
-        $time_end = microtime(true);
-        echo "\e[32mCreated:\e[0m   activity participations ".'('.round(($time_end - $time_start), 2).'s)'.PHP_EOL;
+        // Create quotes
+        $n = 100;
+        $output->task("creating $n quotes", fn () => Quote::factory()->count($n)->create());
 
         // Create AchievementOwnership
```

Comment:
You are creating ActivityParticipations here, not AchievementOwnerships. Also seems a duplicate from line 106? Also on my machine no ActivityParticipations are created.